sql/src/main/java/io/crate/metadata/doc/DocTableInfoBuilder.java

```diff
@@ -109,6 +109,9 @@ private DocIndexMetaData docIndexMetaData() {
             return docIndexMetaData;
         }
         for (String concreteIndice : concreteIndices) {
+            if (IndexMetaData.State.CLOSE.equals(state.getMetaData().indices().get(concreteIndice).getState())) {
+                throw new UnhandledServerException(String.format("Unable to access the partition %s", concreteIndice));
```

Comment:
Is this partition specific error message also thrown if this is a regular table which has it's index closed?